docs/pages/management/admin/labels.mdx

```diff
@@ -280,6 +282,58 @@ computer. Your Teleport user must be authorized to access the server.
    ip-192-168-13-57 ⟵ Tunnel       arch=x86_64,environment=dev,hostname=ip-192-168-13-57
    ```
 
+## Step 4/4. Apply ephemeral labels using `server_info`
+
+<Notice type="note">
+   Applying ephemeral labels is only supported for servers.
+</Notice>
+
+You can apply ephemeral labels to a Teleport instance by creating a `server_info`
+resource for the instance. For a server with name `<name>`, its matching `server_info`
+should be named `si-<name>`.
+
+To add ephemeral labels:
+
+1. Create the file `server_info.yaml` and paste the following into it:
+
+   ```yaml
+   # server_info.yaml
+   kind: server_info
+   metadata:
+      # Replace "example-node" with the name of the node to apply labels to
+      name: si-example-node
+   spec:
+      new_labels:
+         "foo": "bar"
+   ```
+
+   Run the following to create the `server_info` resource:
+
+   ```code
+   $ tctl create server_info.yaml
+   ```
+
+1. Verify that you have added the label by running the following command on your local
+computer. Your Teleport user must be authorized to access the server. Teleport
+applies labels from `server_info` resources gradually to prevent strain on the Auth Service in
+larger clusters, so it may take several minutes for the new labels to appear.
+
+   ```code
+   $ tsh ls
+   ```
+
+   You should see output similar to the following with the `foo` label displayed:
+
+   ```code
+   Node Name        Address        Labels                                                                                                                   
+   ---------------- -------------- ------------------------------------------------------
+   ip-192-168-13-57 ⟵ Tunnel       foo=bar,hostname=ip-192-168-13-57
+   ```
+
+Although ephemeral labels do not persist across restarts, they will be set again
```

Comment:
I don't quite understand what this means.

If you restart Teleport, the `server_info` resource will still exist, so the labels will remain. Isn't this considered persistence?

Reply:
Fair enough. The labels are stored in memory on the server and are lost if the server restarts until the auth server recreates them, so I wanted to make sure that users didn't panic when they restart the server and don't see the labels. I've updated this section to clarify what's going on.